src/main/resources/publishAndTag

```diff
@@ -20,7 +20,7 @@ if [[
 
   # Only publish on oraclejdk8
   if [[ $JAVA_HOME == *"java-8-oracle" ]]; then
-    sbt ++"$TRAVIS_SCALA_VERSION" publishSigned sonatypeRelease
+    sbt ++"$TRAVIS_SCALA_VERSION" 'sonatypeOpen "sbt-slamdata staging"' publishSigned sonatypeRelease
```

Comment:
Isn't `publishSigned` just going to open a new repository in addition to this one?

Reply:
From what I can tell it will use the same repo - at least that what `sonatypeRelease` did when testing. Didn't test `publishSigned` though, as I didn't want to actually publish artifacts.

Reply:
Well, let's give it a shot!